### ISSUE TITLE
[Suggestion] Quick action : show markers parent category in main menu ?

### ISSUE THREAD

**Issue by Arkane-x6** [blish-hud/Pathing#99]
Add an action to the right click markers menu to, open the main menu and reach (and highlight) the parent category

***What do you mean?***
Right click on a marker opens a menu. Click on (new)  *"Show parent category in main menu"*
*eg. the marker has the category `ParentA.ParentAA.ParentAAA`*
It will act like we click on the main menu, hover the ParentA, then the ParentAA, etc.

For example, it would be easier (to me at least) to check if a marker can be toggled or not, or its parent category. Or quickly check which marker is in this category
In short: Have more control of which category I'll (un)check or not
